components/gripper/gripper.go

```diff
@@ -34,6 +34,16 @@ func Named(name string) resource.Name {
 }
 
 // A Gripper represents a physical robotic gripper.
+//
+// Open example:
+//
+//	// Open the gripper.
+//	err := myGripper.Open(context.Background(), nil)
```

Comment:
```suggestion
//	err := myGripperComponent.Open(context.Background(), nil)
```
Should we standardize on appending "Component" to the ends of all component names since that's how the code sample boilerplate initializes them?

Reply:
Feel free to disregard until we hear back about standardization